src/bucket.rs

```diff
@@ -71,4 +82,24 @@ impl Bucket {
     pub fn get_fingerprint_index(&self, fp: Fingerprint) -> Option<usize> {
         self.buffer.iter().position(|e| *e == fp)
     }
+
+    /// Returns all current fingerprint data of the current buffer for storage.
+    pub fn get_fingerprint_data(&self) -> Vec<u8> {
```

Comment:
This could be renamed to an `impl To<Vec<u8>> for Bucket` to have a nice analoguon to `From<[&'a u8]> for Bucket`, I think, if this is not intentional.

Reply:
I'm not sure how to give [this closure](https://github.com/seiflotfy/rust-cuckoofilter/pull/29/files#diff-b4aea3e418ccdb71239b96952d9cddb6R234) the correct type annotations if `get_fingerprint_data` is replaced by `into`. That was the only intention there. If you know how to coerce that, I can update it.

Reply:
Gave it a quick try myself, now I see. Generally, one could annotate the type with `.into::<Vec<u8>>()`, but it does not get as handy as I thought because `into` takes `self` and not `&self`.

Reply:
Yeah, I attempted that too, it seemed like more trouble than it's worth, since you'd be coercing & converting back and forth for some ideal of symmetry, but it wouldn't make it cleaner in the long run&mdash;instead it'll just be less readable.

Reply:
Agreed :)